sbaid/model/results/result_manager.py

```diff
@@ -104,43 +102,31 @@ async def register_result(self, result: Result) -> None:
-                                          result.creation_date_time)
-        await self.__add_all_snapshots_to_database(result)
-
-    async def __add_all_snapshots_to_database(self, result: Result) -> None:
-
+        snapshot_data: list[tuple[str, str, str,
+                                  list[tuple[str, str, str, str, int,
+                                             list[tuple[str, str, int, float, int, int,
+                                                        list[tuple[str, int, float]]]]]]]] = []
         for snapshot in result.snapshots:
-            assert isinstance(snapshot, Snapshot)
-
-            await self.__global_db.add_snapshot(snapshot.id,
-                                                result.id,
-                                                snapshot.capture_timestamp)
-            for cs_snapshot in snapshot.cross_section_snapshots:
-                assert isinstance(cs_snapshot, CrossSectionSnapshot)
-
-                await self.__global_db.add_cross_section_snapshot(cs_snapshot.cs_snapshot_id,
-                                                                  cs_snapshot.snapshot_id,
-                                                                  cs_snapshot.cross_section_id,
-                                                                  cs_snapshot.cross_section_name,
-                                                                  cs_snapshot.b_display)
-
-                for lane_snapshot in cs_snapshot.lane_snapshots:
-                    assert isinstance(lane_snapshot, LaneSnapshot)
-
-                    await self.__global_db.add_lane_snapshot(lane_snapshot.id,
-                                                             lane_snapshot.
-                                                             cross_section_snapshot_id,
-                                                             lane_snapshot.lane,
-                                                             lane_snapshot.average_speed,
-                                                             lane_snapshot.traffic_volume,
-                                                             lane_snapshot.a_display)
-
-                    for vehicle_snapshot in lane_snapshot.vehicle_snapshots:
-                        assert isinstance(vehicle_snapshot, VehicleSnapshot)
-                        await self.__global_db.add_vehicle_snapshot(vehicle_snapshot.
-                                                                    lane_snapshot_id,
-                                                                    vehicle_snapshot.vehicle_type,
-                                                                    vehicle_snapshot.speed)
+            cs_sn_data: list[tuple[str, str, str, str, int,
+                                   list[tuple[str, str, int, float, int, int,
+                                              list[tuple[str, int, float]]]]]] = []
+            for cs_sn in snapshot.cross_section_snapshots:  # type: ignore
```

Comment:
lieber casten als type ignore

Reply:
dann würde ich wo du des brauchst in der for Loop irgendwie 
for obj_cs_sn in ...:
     cs_sn = cast(CrossSectionSnapshot, obj_cs_sn)

machen. weil wenn wir kein Type ignore brauchen sollten wir es auch nicht nutzen IMO